examples/sqlite3.roc

```diff
@@ -0,0 +1,45 @@
+app "echo"
+    packages { pf: "../platform/main.roc" }
+    imports [
+        pf.Stdout,
+        pf.Task.{ Task },
+        pf.Http.{ Request, Response },
+        pf.SQLite3,
+        pf.Env,
+    ]
+    provides [main] to pf
+
+main : Request -> Task Response []
+main = \_ ->
+
+    # Read DB_PATH environment variable
+    maybeDbPath <- Env.var "DB_PATH" |> Task.attempt
+
+    # Query todos table
+    rows <-
+        SQLite3.execute {
+            path: maybeDbPath |> Result.withDefault "<DB_PATH> not set on environment",
+            query: "SELECT id, task FROM todos WHERE status = :status;",
+            bindings: [{ name: ":status", value: "completed" }],
```

Comment:
I dunno if this has been discussed elsewhere, but it should be possible to have the bindings look like `{ status: "completed" }` and then have an `Encoding` implementation which takes care of translating them to the equivalent of this! 😃